examples/example-bigquery-to-bigquery-update-partition-filter-enabled-cdap-data-pipeline.json

```diff
@@ -0,0 +1,105 @@
+        "name": "cdap-data-pipeline",
+        "version": "6.1.1",
+        "scope": "SYSTEM"
+    },
+    "description": "Data Pipeline Application",
+    "name": "example-bigquery-to-bigquery-update-partition-filter-enabled",
+    "config": {
+        "resources": {
+            "memoryMB": 2048,
+            "virtualCores": 1
+        },
+        "driverResources": {
+            "memoryMB": 2048,
+            "virtualCores": 1
+        },
+        "connections": [
+            {
+                "from": "BigQuery",
+                "to": "BigQuery2"
+            }
+        ],
+        "comments": [],
+        "postActions": [],
+        "properties": {},
+        "processTimingEnabled": true,
+        "stageLoggingEnabled": false,
+        "stages": [
+            {
+                "name": "BigQuery",
+                "plugin": {
+                    "name": "BigQueryTable",
+                    "type": "batchsource",
+                    "label": "BigQuery",
+                    "artifact": {
+                        "name": "google-cloud",
+                        "version": "0.15.0-SNAPSHOT",
+                        "scope": "USER"
+                    },
+                    "properties": {
+                        "project": "auto-detect",
+                        "serviceFilePath": "auto-detect",
+                        "dataset": "${sourceDataset}",
+                        "table": "${sourceTable}",
+                        "referenceName": "BigQuerySource",
+                        "schema": "${outputSchema}"
+                    }
+                },
+                "outputSchema": [
+                    {
+                        "name": "etlSchemaBody",
+                        "schema": "${outputSchema}"
+                    }
+                ]
+            },
+            {
+                "name": "BigQuery2",
+                "plugin": {
+                    "name": "BigQueryTable",
+                    "type": "batchsink",
+                    "label": "BigQuery2",
+                    "artifact": {
+                        "name": "google-cloud",
+                        "version": "0.15.0-SNAPSHOT",
+                        "scope": "USER"
+                    },
+                    "properties": {
+                        "project": "auto-detect",
+                        "serviceFilePath": "auto-detect",
+                        "operation": "update",
+                        "truncateTable": "false",
+                        "allowSchemaRelaxation": "false",
+                        "location": "US",
+                        "createPartitionedTable": "false",
+                        "partitionFilterRequired": "false",
+                        "schema": "${inputSchema}",
+                        "referenceName": "BigQuerySinkExample",
+                        "table": "${sinkTable}",
+                        "relationTableKey": "${tableKey}",
+                        "partitionFilter": "${partitionFilter}",
```

Comment:
how does the filter look like in this case?

Reply:
This represents the partition elimination for the update/upsert operations on existing tables where partition filter is enabled. Examples:
1. If you have partitioned table by time:
`_PARTITIONTIME > "2020-01-01"`
`_PARTITIONTIME > "2020-01-01" AND _PARTITIONTIME < "2020-03-01"`

2. If you have partitioned table partitioned by Integer value (e.x. gameId):
`gameId < 5`
`gameId > 1 AND gameId < 7`